src/ol/source/GeoTIFF.js

```diff
@@ -108,6 +108,8 @@ function readRGB(preference, image) {
 const STATISTICS_MAXIMUM = 'STATISTICS_MAXIMUM';
 const STATISTICS_MINIMUM = 'STATISTICS_MINIMUM';
 
+const defaultTileSize = 256;
```

Comment:
Is `256` an arbitrary value, or did you choose this purposely over e.g. `512`?

Reply:
We probably need to see how this works out in practice.  If we make it too large, and the image is very wide, it could take a long time to load and parse - since the range requests will be for batches of rows that span the whole image width.